assets/clients/README.md

```diff
@@ -1,136 +1,292 @@
 
 ---
 
-## `icons` Section
+#### 1.2 `downloads` List
+
+Each item in `downloads` describes a button/badge/link for where to get the client. The generator has multiple **type** handlers, each producing a specific style of link or shield badge.
+
+Each `Download` object has a **`type`** key (required) to determine which of the generator’s “renderer” structs to use:
+
+| `type` Value    | Additional Fields                                          | Purpose / Example Output                                                                                                                          |
+|-----------------|------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`icon`**      | `icon`, `url`                                             | Renders an icon from `assets/clients/icons/<icon>.png` linked to `url`. Not typically used if you want dynamic badges like shield.io, but can embed a static icon. |
+| **`text`**      | `text`, `url`                                             | Renders a simple **text link** with `[text](url)` format.                                                                                        |
+| **`github`**    | `owner`, `repo`, optional `url`                           | Renders a GitHub total downloads badge (`![](https://img.shields.io/github/downloads/owner/repo/total)`). Defaults the link to GitHub releases unless `url` is provided. |
+| **`flathub`**   | `package`, optional `url`                                 | Renders a Flathub downloads badge for a specific Flatpak package.                                                                                |
+| **`docker`**    | `user`, `repo`, optional `url`                            | Renders a Docker Hub pulls badge.                                                                                                                |
+| **`shield`**    | `label`, `content`, `icon`, `color`, **`url`**            | Builds a custom shield.io badge. All fields except `url` are optional, letting you control label text, icon, color, etc.                          |
+| **`gitlab`**    | `owner`, `repo`, optional `url`                           | Renders a GitLab badge that reuses the `shield` logic under the hood, with a default GitLab logo.                                                |
+| **`demo`**      | **`url`**                                                 | Renders a “Demo” button shield linking to the `url`.                                                                                             |
+| **`app-store`** | `id`, optional `url`                                      | Renders an App Store shield linking to `https://apps.apple.com/app/id<id>` by default.                                                           |
```

Comment:
Is this a typo?
```suggestion
| **`app-store`** | `id`, optional `url`                                      | Renders an App Store shield linking to `https://apps.apple.com/app/<id>` by default.                                                           |
```

Reply:
nope, e.g. https://apps.apple.com/de/app/jellyflix/id6476043683